platform/android/export/export_plugin.cpp

```diff
@@ -2225,6 +2225,9 @@ String EditorExportPlatformAndroid::get_apksigner_path(int p_target_sdk, bool p_
 }
 
 bool EditorExportPlatformAndroid::has_valid_export_configuration(const Ref<EditorExportPreset> &p_preset, String &r_error, bool &r_missing_templates, bool p_debug) const {
+	#ifdef MODULE_MONO_ENABLED
+		r_error += TTR("Exporting to Android when using C#/.NET is experimental.") + "\n";
+	#endif
```

Comment:
Move this block after the `err` declaration and assign to `err` instead, otherwise the message will be replaced when there are other errors.